analyzer/codechecker_analyzer/analyzers/clangsa/ctu_autodetection.py

```diff
@@ -262,4 +262,5 @@ def is_ctu_capable(self):
         if not tool_path:
             return False
 
-        return invoke_binary_checked(tool_path, ['-version']) is not False
+        return invoke_binary_checked(tool_path, ['-version'], self.environ) \
```

Comment:
There is another usage of this function:
https://github.com/Ericsson/codechecker/blob/76d1ea88faa9467563a7dc1ce28d2b03ee77c5d7/analyzer/codechecker_analyzer/analyzers/clangsa/ctu_autodetection.py#L133-L134

Do not we need to forward the `self.environ` to this function call too?